cmd/zt_copy_s2smigration_test.go

```diff
@@ -1329,7 +1329,7 @@ func (s *cmdIntegrationSuite) TestCopyWithDFSResource(c *chk.C) {
 		c.Assert(err, chk.IsNil)
 
 		// validate that the right number of transfers were scheduled
-		c.Assert(len(mockedRPC.transfers), chk.Equals, 1)
+		c.Assert(len(mockedRPC.transfers), chk.Equals, 2)
```

Comment:
is this to account for the root as well now?

Reply:
Because DFS is no longer treated as blob for S2S, folders are included by default. This change tracks the additional folder.